### PR TITLE
sync: Completely parallelize the initial payload

### PR DESCRIPTION
**Pull request by smarterclayton**
Increase parallism to maximum tasks, increase backoff, wait longer for cluster operator, and generally try to load everything at the same time.

This results in a significant reduction in boot time and allows us to keep run-levels in the payload in the desired update order, not the desired install order.

Builds on top of #133

### CODE REVIEW

**Comment by smarterclayton** [comment]
Console continues to take the longest - last run was 11m, monitoring finished at 8m30 and console took just about 5m total.

**Comment by smarterclayton** [comment]
Ok, looks like parallel installation was able to break at least openshift-apiserver: https://openshift-gce-devel.appspot.com/build/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/136/pull-ci-openshift-cluster-version-operator-master-e2e-aws/459

/retest

**Comment by smarterclayton** [comment]
So I've seen one break out of 6, which is about what mainline is.  I think this is ready for review @abhinavdahiya

File: pkg/payload/task_graph_test.go [hunk 1]
```diff
@@ -233,6 +233,143 @@ func TestByNumberAndComponent(t *testing.T) {
...
+			want: [][]*TaskNode{
+				{
+					&TaskNode{Tasks: tasks("a")},
+					&TaskNode{Tasks: tasks("0000_01_x-y-z_file1")},
+					&TaskNode{Tasks: tasks("c")},
+				},
+			},
+		},
+		{
+			tasks: tasks("0000_01_x-y-z_file1", "0000_01_x-y-z_file2"),
```
Comment (abhinavdahiya) [inline pkg/payload/task_graph_test.go]: this is same as https://github.com/openshift/cluster-version-operator/pull/136/commits/fee2d0687137545ca8f8e84e91ec7b2429887c02#diff-6c3be3194acbe263715006cdee7e9442R276 `multiple grouped items in single node`

**Comment by abhinavdahiya** [comment]
This looks LGTM.
trying to how much speed up we get..
from this PR:
```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/136/pull-ci-openshift-cluster-version-operator-master-e2e-aws/468/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-13T15:39:00Z",
  "completed": "2019-03-13T15:50:10Z"
}
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/136/pull-ci-openshift-cluster-version-operator-master-e2e-aws/464/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-13T00:24:35Z",
  "completed": "2019-03-13T00:35:22Z"
}
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_cluster-version-operator/136/pull-ci-openshift-cluster-version-operator-master-e2e-aws/458/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-12T05:04:14Z",
  "completed": "2019-03-12T05:18:23Z"
}
```
on an average 11 min to level.

compared to CI runs on installer (without this change):
```console
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1407/pull-ci-openshift-installer-master-e2e-aws/4464/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-12T19:35:32Z",
  "completed": "2019-03-12T19:54:15Z"
}
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1407/pull-ci-openshift-installer-master-e2e-aws/4475/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-13T15:29:54Z",
  "completed": "2019-03-13T15:48:44Z"
}
$ curl -s https://storage.googleapis.com/origin-ci-test/pr-logs/pull/openshift_installer/1393/pull-ci-openshift-installer-master-e2e-aws/4470/artifacts/e2e-aws/clusterversion.json | jq '.items[] | select (.metadata.name == "version") | .status.history[0] | {started: .startedTime, completed: .completionTime}'
{
  "started": "2019-03-13T02:55:20Z",
  "completed": "2019-03-13T03:13:48Z"
}
```
so on an average 19min to level.

This adds a significant speed up. :100: 

/approve

**Pull request merged**